oss-internship-2020/libtiff/test/check_tag.cc

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "check_tag.h"  // NOLINT(build/include)
+
+using ::sapi::IsOk;
+using ::testing::Eq;
+using ::testing::IsTrue;
+
+void CheckShortField(TiffApi& api, sapi::v::RemotePtr& tif, const ttag_t field,
+                     const uint16_t value) {
+  sapi::v::UShort tmp(123);
```

Comment:
if the intention is to fail the test if the getter does not update the value use a initial value of e.g. `value+1`, otherwise just use the default ctor.